catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/FieldDescriptors.js

```diff
@@ -176,8 +95,8 @@ define(['underscore'], function (_) {
                 },
                 Person: {
                     Name: {
-                        displayName: 'Contact Designation',
-                        description: 'Contact Designation',
+                        displayName: 'Contact Title',
+                        description: 'Contact Title',
```

Comment:
I know these seem obvious, but maybe as part of that potential tech debt we update any descriptions that matches the field name.